src/support/timestamp.c

```diff
@@ -43,15 +43,17 @@ __wt_time_point_to_string(
 char *
 __wt_time_window_to_string(WT_TIME_WINDOW *tw, char *tw_string)
 {
-    char ts_string[4][WT_TS_INT_STRING_SIZE];
+    char ts_string[6][WT_TS_INT_STRING_SIZE];
 
     WT_IGNORE_RET(__wt_snprintf(tw_string, WT_TIME_STRING_SIZE,
```

Comment:
We need to increase the size of the `WT_TIME_STRING_SIZE` to 6 timestamps and 4 unit64 values.